sounds/samples/README.md

```diff
@@ -3,7 +3,13 @@ How to add a new sample sound
 
 * Find a sample under a free/libre open-source (FOSS) license.
 
-* The sample has to have only one note (pitch) through out the audio and the volume has to be 50 . The longer the sample, the better it is.
+### Checklist for Sample
+
+* [ ] Only one pitch (no melodic movement)
+
+* [ ] After tuning the synth, use your ear and ask yourself "does this pitch sound the similar" (Audacity can be used to some extent for fundamental pitch)[Audacity](https://manual.audacityteam.org/man/change_pitch.html).
```

Comment:
Can you please clarify: "does this pitch sound the similar"? I think you don't need "the". But, similar to what?

Reply:
It means constant frequency throughout the sample.  In issue #1485 he mentioned about English Horn and vibraphone playing two notes.

Reply:
So perhaps you should say: "does this sound have a constant frequency throughout the sample?"